MOBILE_API.xml

```diff
@@ -8236,12 +8236,15 @@
                 This parameter is filled for supporting OEM proprietary data exchanges.
             </description>
         </param>
-        <param name="url" type="String" maxlength="1000" mandatory="false">
+        <param name="url" type="String" mandatory="false" since="6.1">
```

Comment:
```suggestion
        <param name="url" type="String" mandatory="false" since="7.0">
```
RPC spec version has been bumped to 7.0 https://github.com/smartdevicelink/rpc_spec/pull/264

Reply:
@ShobhitAd 
Fixed in:https://github.com/smartdevicelink/rpc_spec/pull/225/commits/d2bb2bc092224bc395eda1da78bad4a29434bd99